packages/roosterjs-editor-plugins/lib/Paste/Paste.ts

```diff
@@ -180,7 +191,11 @@ export default class Paste implements EditorPlugin {
         }, ChangeSource.Paste);
     }
 
-    private applyTextFormat(node: Node, format: DefaultFormat) {
+    private applyFormatting = (format: DefaultFormat) => (element: HTMLElement) => {
+        applyFormat(element, format);
+    }
+
+    private applyToElements(node: Node, callback: (element: HTMLElement) => void) {

```

Comment:
docstring pls (callback is not a descriptive arg name -- an explanation or a better name would be needed).